exp/rendering-architecture.md

```diff
@@ -15,6 +17,16 @@ For AMD and Intel GPUs, Anbox Cloud uses the [Mesa driver](https://www.mesa3d.or
 
 See [Component versions](https://anbox-cloud.io/docs/component-versions) to refer to the actual version supported for any particular Anbox Cloud release.
 
+# Supported platforms
+
+For rendering, you can use the `swrast` or the `null` platforms depending on your requirements.
+
+`swrast` is a software rasterization platform, which is a rendering implementation of the Mesa driver with support for LLVMpipe. It can be utilised for use cases that require a visual output without a GPU. When you launch an Anbox container with the `swrast` platform using the `amc launch -p swrast <app_id>` command, the rendering pipeline uses this platform irrespective of any available GPUs. The rendering pipe for the `swrast` or `null` platform is not different than the one for the `webrtc` platform with NVIDIA GPU support except for the fact that it's irrespective of any available GPUs. To know more about this implementation, see [LLVMpipe](https://docs.mesa3d.org/drivers/llvmpipe.html).
```

Comment:
` the rendering pipeline uses this platform irrespective of any available GPUs`
Can we drop this sentence as the following one cover the point already?

Reply:
@adglkh 

> When you launch an Anbox container with the `swrast` platform using the `amc launch -p swrast <app_id>` command, the rendering pipeline uses this platform irrespective of any available GPUs.

This statement also informs the user they can launch a container with the particular command. Are you fine with dropping this entire statement? Or just the last part that talks about the pipeline irrespective of GPUs?

Reply:
I feel these two sentences are a bit repetitive as they're both explaining the point that the rendering pipeline uses this platform irrespective of any available GPUs. I vote for dropping this entire statement and only keeping `The rendering pipe for the `swrast` or `null` platform...`
what do you think?

Reply:
@adglkh Agree, done.